toolkit/themes/shared/design-system/config.js

```diff
@@ -0,0 +1,64 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* eslint-env node */
+
+const StyleDictionary = require("style-dictionary");
+const { fileHeader } = StyleDictionary.formatHelpers;
+
+/**
+ * CSS formatter that converts tokens that have a light and dark value
+ * to a light-dark() function. If we try to use the built-in CSS formatter,
+ * we will lose the "darkValue" or any other property that isn't "value".
+ */
+function lightDarkFormatter(args) {
+  let dictionary = args.dictionary;
+  let file = args.file;
+  const selector = args.options.selector ? args.options.selector : `:root`;
+  const indentation = args.options.indentation ? args.options.indentation: 2;
+  let modifiedDictionary = dictionary.allTokens.map(token => {
+    let value = JSON.stringify(token.value);
+    // the `dictionary` object now has `usesReference()` and
+    // `getReferences()` methods. `usesReference()` will return true if
+    // the value has a reference in it. `getReferences()` will return
+    // an array of references to the whole tokens so that you can access their
+    // names or any other attributes.
+    if (dictionary.usesReference(token.original.value) && token.original.darkValue) {
```

Comment:
Does this need to check that the light value uses a reference? Something like this seems to throw away the `light-dark` now

```
"brand": {
  "value": "black",
  "darkValue": "{color.gray.05}"
}
```
```suggestion
    if (token.original.darkValue) {
```

Trying this change locally means that the build will fail, which seems fine if we don't support the case as authored.